test/scripts/prepare_agent_config.sh

```diff
@@ -1,24 +1,6 @@
 #!/usr/bin/env bash
 
-SCRIPT_DIR="$(dirname "$(readlink -f "$0")")"
-IP=$("${SCRIPT_DIR}"/get_ext_ip.sh)
-OUTFILE=bin/agent/etc/flightctl/config.yaml
-mkdir -p bin/agent/etc/flightctl/certs
 cp ~/.flightctl/certs/ca.crt bin/agent/etc/flightctl/certs/ca.crt
 cp ~/.flightctl/certs/client-enrollment.{crt,key} bin/agent/etc/flightctl/certs/
 
-echo "Creating agent config in ${OUTFILE}"
-tee ${OUTFILE} <<EOF
-enrollment-service:
-  authentication:
-    client-certificate: certs/client-enrollment.crt
-    client-key: certs/client-enrollment.key
-  service:
-    certificate-authority: certs/ca.crt
-    server: https://agent-api.${IP}.nip.io:7443
-  enrollment-ui-endpoint: https://ui.${IP}.nip.io:8080
-grpc-management-endpoint: grpcs://agent-grpc.${IP}.nip.io:7444
-spec-fetch-interval: 0m10s
-status-update-interval: 0m10s
-tpm-path: /dev/tpmrm0
-EOF
+./bin/flightctl certificate request > bin/agent/etc/flightctl/config.yaml
```

Comment:
```suggestion
echo Requesting enrollment enrollment certificate/key and config for agent =====
./bin/flightctl certificate request | tee bin/agent/etc/flightctl/config.yaml
```

Just a suggestion, it will help us see what's going on.